packages/model-viewer/src/test/model-viewer-base-spec.ts

```diff
@@ -338,10 +338,10 @@ suite('ModelViewerElementBase', () => {
 
       test('sets a model within viewport to be visible', async () => {
         await until(() => {
-          return elements[0][$scene].visible;
+          return elements[2][$scene].visible;
```

Comment:
Why this change?

Reply:
Because we reversed the order that the elements are inserted to the document.